script.js

```diff
@@ -0,0 +1,161 @@
+mainSpeakers.style.display = 'grid';
+
+const descriptionStyle = {
+  position: 'relative',
+  marginLeft: '10px',
+  marginBottom: '16px',
+  fontFamily: 'Lato',
+  fontSize: '0.8rem',
+};
+
+// Checker background style
+const chessStyle = {
+  position: 'absolute',
+  source: 'icons/checkers.svg',
+  width: '65px',
+  zIndex: '1',
+  top: '-15px',
+  left: '11px',
+};
+
+const speakersImageStyle = {
+  width: '120px',
+  marginLeft: '20px',
+  zIndex: '1',
+  position: 'relative',
+};
+
+function gridify() {
+  if (window.innerWidth >= 768) {
+    mainSpeakers.style.gridTemplateColumns = '3fr 3fr';
+  } else if (window.innerWidth <= 768) {
+    mainSpeakers.style.gridTemplateColumns = '1fr';
+  }
+}
+
+const body = document.querySelector('body');
+body.addEventListener('click', gridify);
+
+const inc = 1;
+function main(count) {
+  for (let i = 0; i < count; i += inc) {
+    const overAll = document.createElement('div');
+    overAll.style.display = 'flex';
+    overAll.style.alignItems = 'center';
+    mainSpeakers.appendChild(overAll);
+
+    const pictureWrapper = document.createElement('div');
+    pictureWrapper.style.position = 'relative';
+    pictureWrapper.style.marginBottom = '30px';
+    overAll.appendChild(pictureWrapper);
+
+    // The info about the speaker
+    const descriptionCon = document.createElement('div');
+    descriptionCon.style.marginBottom = '30px';
+    overAll.appendChild(descriptionCon);
+
+    const chess = document.createElement('img');
+    chess.setAttribute('class', 'chess');
+    chess.src = chessStyle.source;
+    Object.assign(chess.style, chessStyle);
+    pictureWrapper.appendChild(chess);
+
+    const speaker = document.createElement('img');
+    speaker.src = speakers[i].image;
+    Object.assign(speaker.style, speakersImageStyle);
+    pictureWrapper.appendChild(speaker);
+
+    const name = document.createElement('h3');
+    Object.assign(name.style, descriptionStyle);
+    name.innerHTML = speakers[i].name;
+    name.style.fontWeight = '600';
+    descriptionCon.appendChild(name);
+
+    const description = document.createElement('h6');
+    Object.assign(description.style, descriptionStyle);
+    description.style.color = '#ec5242';
+    description.style.marginTop = '-10px';
+    description.style.fontWeight = '600';
+    description.innerHTML = speakers[i].description;
+    descriptionCon.appendChild(description);
+
+    const expand = document.createElement('h6');
+    Object.assign(expand.style, descriptionStyle);
+    expand.style.color = '#272a31';
+    expand.innerHTML = speakers[i].expanded;
+    descriptionCon.appendChild(expand);
+  }
+}
+
+const seemore = document.querySelector('#seemore');
+seemore.addEventListener('click', () => {
+  mainSpeakers.innerHTML = '';
+  seemore.style.opacity = '0';
+  main(6);
+});
+
+if (window.innerWidth >= 768) {
+  window.onload = main(4);
+  mainSpeakers.style.gridTemplateColumns = '1fr 1fr';
+} else {
+  window.onload = main(2);
```

Comment:
In order to stick to Cindy's design, I suggest that you make sure the mobile version of the home page ends at the show more button.
Let's make fix the button too to avoid any bugs

your design
![image](https://user-images.githubusercontent.com/60612329/226094586-ffe16076-e85a-4e33-83c9-a440af057b7c.png)